.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,84 @@
+---
+name: Problem report
+about: Create an extensive report to help us document a problem
+
+---
+<!--- Please fill out this template to the best of your ability. You can always edit this issue once you have created it. -->
+## Bug report
+### Describe the bug
+Here is a clear and concise description of what the problem is:
+<!--- Provide a more detailed introduction to the issue itself, and why you consider it to be a bug -->
+<!--- A bug report that is not clear will be closed -->
+<!--- Put your text below this line -->
+
+
+
+## Expected Behavior
+Here is a clear and concise description of what was expected to happen:
+<!--- Tell us what should happen -->
+<!--- Put your text below this line -->
+
+
+
+## Actual Behavior
+<!--- Tell us what happens instead -->
+<!--- Put your text below this line -->
+
+
+
+## Possible Fix
+<!--- Not obligatory, but suggest a fix or reason for the bug -->
+<!--- Put your text below this line -->
+
+
+
+### To Reproduce
+Steps to reproduce the behavior:
+<!--- Provide a link to a live example, or an unambiguous set of steps to -->
+<!--- reproduce this bug. Include code to reproduce, if relevant -->
+<!--- Put your text below this line -->
+1. 
+2.
+3.
+
+
+### Debuglog
+<!--- Put your text below this line -->
+<!--- A debuglog is always mandatory when creating an issue. Provide one! -->
```

Comment:
A reminder that at least debug logging should be enabled for the log is needed, and component-specific logging too if applicable to the issue.

A link to the debug log wiki page may well be worth including too if possible.

Reply:
We shouldn't start replicating the entire wiki page of how to report a bug imo.
I'll put in the link at the top. If tickets are not clear enough we should either ask to add extra information or plain close them as invalid

Reply:
The link should be enough, if it gets followed. 

But from personal experience of troubleshooting stuff at work, I always find the more detailed request you make for information, the smoother, more quick and less frustrating things are all round.

I would at least add "debug enabled" to the debug log request though.